includes/rest/class-outbox-controller.php

```diff
@@ -175,10 +175,22 @@ public function get_items( $request ) {
 			'generator'    => 'https://wordpress.org/?v=' . get_masked_wp_version(),
 			'actor'        => $user->get_id(),
 			'type'         => 'OrderedCollection',
-			'totalItems'   => $outbox_query->found_posts,
+			'totalItems'   => 0,
 			'orderedItems' => array(),
 		);
 
+		$post_types = \get_option( 'activitypub_support_post_types', array( 'post' ) );
+
+		if ( $user_id > Actors::BLOG_USER_ID ) {
+			$count_posts            = \count_user_posts( $user_id, $post_types, true );
+			$response['totalItems'] = \intval( $count_posts );
+		} else {
+			foreach ( $post_types as $post_type ) {
+				$count_posts             = \wp_count_posts( $post_type );
```

Comment:
I assume both `count_user_posts()` and `wp_count_posts()` include unfederated posts, not sure how you feel about that?

Reply:
That was what we used before. I fear that custom queries would make the request unnecessarily slow!?